content-src/experiments/no-more-404s.yaml

```diff
@@ -13,7 +13,7 @@ contribute_url: 'https://github.com/internetarchive/FirefoxNoMore404s'
 bug_report_url: 'https://github.com/internetarchive/FirefoxNoMore404s/issues/new'
 discourse_url: 'https://discourse.mozilla-community.org/c/test-pilot/nomore404s'
 privacy_notice_url: 'https://github.com/internetarchive/FirefoxNoMore404s/blob/master/docs/metrics.md'
-measurements: "In addition to the <a href=\"https://testpilot.firefox.com/privacy\" target=\"_blank\">data</a> collected by all Test Pilot experiments, here are the key things you should know about what is happening when you use No More 404s:\r\n<ul>\r\n<li>We collect basic usage on how many times you encounter a Page Not Found error (code 404), how many times a cached version of that page exists from Archive.org, and how many times you choose to view the cached version.</li>\r\n<li>To provide cached versions of pages, we send 404 error page URLs to Archive.org. <a href=\"https://archive.org/\" target=_blank\">Archive.org</a> discloses its privacy policy <a href=\"https://archive.org/about/terms.php\" target=\"blank\">here</a>.</li>\r\n<li>We do not  collect  URLs of the pages you request or the URLs we send to Archive.org.</li>\r\n<li>We may share survey results you submit to us and aggregated telemetry data related to this experiment with the Internet Archive.</li>\r\n</ul>"
+measurements: "In addition to the <a href=\"/privacy\" target=\"_blank\">data</a> collected by all Test Pilot experiments, here are the key things you should know about what is happening when you use No More 404s:\r\n<ul>\r\n<li>We collect basic usage on how many times you encounter a Page Not Found error (code 404), how many times a cached version of that page exists from Archive.org, and how many times you choose to view the cached version.</li>\r\n<li>To provide cached versions of pages, we send 404 error page URLs to Archive.org. <a href=\"https://archive.org/\" target=_blank\">Archive.org</a> discloses its privacy policy <a href=\"https://archive.org/about/terms.php\" target=\"_blank\">here</a>.</li>\r\n<li>We do not  collect  URLs of the pages you request or the URLs we send to Archive.org.</li>\r\n<li>We may share survey results you submit to us and aggregated telemetry data related to this experiment with the Internet Archive.</li>\r\n</ul>"
```

Comment:
It'd be nice if we can do some multiline string formatting or something to avoid all the single+double quote escaping.